packages/engine/Source/Scene/CreditDisplay.js

```diff
@@ -351,12 +341,12 @@ function CreditDisplay(container, delimiter, viewport) {
   lightboxCredits.appendChild(creditList);
 
   const cesiumCreditContainer = document.createElement("div");
-  cesiumCreditContainer.className = "cesium-credit-logoContainer";
+  cesiumCreditContainer.className = "cesium-credit-cesiumCreditContainer";
```

Comment:
We may want to avoid changing the class names, as it would be a breaking change for anyone who may have applied custom styles. If there is a good reason to change the class names, it should be listed under "Breaking Changes".

Reply:
I changed this for clarity because I was _really_ confused when I first started looking at this and wondering why the Google "logo" was not in the logo container. I wasn't thinking of external devs trying to style based on this.

In general I think I would argue that this is an "internal feature" and should _not_ be used or relied on for styling in other applications. We allow you to specify the container this component is added to but I would assume we are the only ones that modify the rendering of the internal parts like this. Similar to how we treat private functions in the API that we say can break at any time without notice.

I can change it back for now but I think it might be worth having a larger discussion about this and the components we provide through the Widgets and what people can expect to have "access" to outside of this PR.

Reply:
`cesium-credit-textContainer` changes too. Can you restore that as well?